QgisModelBaker/libqgsprojectgen/dataobjects/project.py

```diff
@@ -137,7 +137,7 @@ def create(self, path: str, qgis_project: QgsProject):
                     'ShowOpenFormButton': False,
                     'AllowNULL': True,
                     'AllowAddFeatures': False,
-                    'FilterExpression': "\"topic\" = '{}'".format( referencing_layer.model_topic_name ),
+                    'FilterExpression': "\"topic\" = @interlis_topic",
```

Comment:
@m-kuhn This does not work. I think I know why. The expression is done on the referenced layer and the variable is from the current (referencing) layer. Do you have an idea or should we just keep the "hard" entered topic name there?